app/backplane/radio_module/src/c_lora_transmit_tenant.cpp

```diff
@@ -12,7 +12,7 @@ void CLoraTransmitTenant::PostStartup() {
 }
 
 void CLoraTransmitTenant::Run() {
-    NRadioModuleTypes::RadioBroadcastData data{};
+    NTypes::RadioBroadcastData data{};
```

Comment:
rip radio mod. its not special enough to get its own namespace. Was there just not enough in the namespace to make it worth an entire namespace?

Reply:
I just want to keep things standard to NTypes. Also, foreseeing the future yaml infrastructure